man/man1/nosetests.1

```diff
@@ -0,0 +1,581 @@
+.\" Man page generated from reStructuredText.
```

Comment:
This file was probably added to the commit by mistake, huh?

Reply:
Uuuuh yeah...